addon/adapters/class.js

```diff
@@ -1,2 +1 @@
-import BaseAdapter from './-addon-docs';
```

Comment:
I don't think we need to keep the `extends` Exporting the default one I think should be fine, since they are all the same. I am unsure why they were extended before.

Reply:
agree @miguelcobain!